changelog.yaml

```diff
@@ -21,6 +21,27 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - type: enhancement
+    impact: minor
+    title: Introduce network profiles
+    description: >-
```

Comment:
We need to change all `>-` to `|-`. Don't touch the existing ones to avoid conflicts.